static/_redirects

```diff
@@ -98,7 +98,10 @@
 /docs/odbc_connector/12.2.0.1/*  /docs/odbc_connector/latest/  301
 
 # BigAnimal
-/docs/edbcloud/* /docs/biganimal/:splat  301
+/docs/edbcloud/* /platform-compatibility:splat  301
```

Comment:
Don't think you want to redirect all edbcloud links to this... Belongs on next line?